examples/structured_data/movielens_recommendations_transformers.py

```diff
@@ -0,0 +1,606 @@
+"""
+Title: Recommendations with Transformer
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/12/30
+Last modified: 2020/12/30
+Description: Using the Behavior Sequence Transformer model for on Movielens dataset.
+"""
+
+"""
+## Introduction
+
+This example demonstrates the [Behavior Sequence Transformer (BST)](https://arxiv.org/abs/1905.06874)
+model, by Qiwei Chen et al., using the [Movielens dataset](https://grouplens.org/datasets/movielens/).
+The BST model leverages the sequential behaviour of the users in watching and rating movies,
+as well as user profile and movie features, to predict the rating of the user to a target movie.
+
+More precisely, the BST model aims to predict the rating of a target movie by accepting
```

Comment:
Suggestion: remove words that don't change the meaning of the sentence:

```suggestion
The BST model aims to predict the rating of a target movie by accepting
```